bolt-simple-taproot.md

```diff
@@ -0,0 +1,1141 @@
+corresponding to its secret integer, thus resulting in a 66-byte value:
+
+```
+point_1 || point_2
+```
+
+For nonce generation, we refer to the `NonceGen` algorithm defined in the
+`musig2` BIP. We only specify the set of required arguments, though a signer can also opt to specify the additional
+argument in order to strengthen their nonce.
+
+#### Nonce Handling
+
+For commitment transactions, Due to the asymmetric state of the current
+revocation channels, two sets of public nonces need to be exchanged by each
+party: one for the local commitment, and one for the remote commitment.
+
+The nonce for the local commitment MUST be sent a priori, thus enabling the
+counterparty to create a valid partial signature.
+
+The nonce for the remote commitment _could_ be sent a priori, too, but for the
+purposes of simplicity and disambiguation, MUST be sent alongside the partial
+signature. To that end, we will create a new data type that will be reused in
+multiple TLV extensions described further down in this document.
+
+The exception to this is the co-op close flow: as there's only a single message
+to sign, we only require a single pair of nonces. These nonces can also be sent
+right as the co-op flow begins.
+
+#### Signing
+
+Once public nonces have been exchanged, the `NonceAgg` algorithm is used to
+combine the public nonces into the aggregate nonce which will be a part of the
+final signature.
+
+The `Sign` method is used to generate a valid partial signature, with the
+`PartialSigVerify` algorithm used to verify each signature.
+
+Once all partial signatures are known, the `PartialSigAgg` algorithm is used to
+aggregate the partial signature into a final, valid [BIP
+340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) Schnorr
+signature.
+
+##### Partial Signature Encoding
+
+A standard `musig2` partial signature is simply the encoding of the `s` value,
+which is a 32-byte-element modulo the order of the group.
+
+Throughout this specification, we will be using a custom type,
+`PartialSignatureWithNonce`, comprised of the aforementioned `s` value, along
+with the 66-byte-encoding of the public nonce used to sign remote commitments:
+
+```
+s || point_1 || point_2
+```
+
+## Design Overview
+
+With the preliminaries out of the way, we provide a brief overview of the
+Simple Taproot Channel design.
+
+The multisig output becomes a single P2TR key, with `musig2` key aggregation
+used to combine two keys into one.
+
+For commitment transactions, we inherit the anchor output semantics, meaning
+there are two outputs used for CPFP, with all other scripts inheriting a `1
+CSV` restriction.
+
+The local output of the commitment transaction uses the revocation key as the
+taproot internal key, and commits to a single script path of the delay script.
+
+The remote output of the commitment transaction uses the `combined_funding_key`
```

Comment:
If the `combined_funding_key` is never used iiuc the new flow, I think it would be better to make it a NUMS point as described in bip340 or bip341 iirc. Otherwise, I would be worried about future uprades where APO is introduced and someone can re-bind a `combined_funding_key` spent on this output, double-spending the remote output.

Reply:
Yep, latest version uses a NUMs point now, we had one, then went back to it.